Code/Editor/EditorModularViewportCameraComposer.h

```diff
@@ -52,5 +62,9 @@ namespace SandboxEditor
         AZStd::shared_ptr<AzFramework::FocusCameraInput> m_orbitFocusCamera;
 
         AzFramework::ViewportId m_viewportId;
+
+        float m_defaultOrbitOpacity = 0.0f; //!< The default orbit axes opacity (to fade in and out).
+        AZ::Vector3 m_defaultOrbitPoint = AZ::Vector3::CreateZero(); //!< The default orbit point.
```

Comment:
nit: `The default orbit point.` comment doesn't provide much additional info, the member name already covers it.

Reply:
I'll update the comment to be more informative